test/integration/elsa/consumer/worker_test.exs

```diff
@@ -94,8 +108,9 @@ defmodule Elsa.Consumer.WorkerTest do
       )
 
     Patiently.wait_for(fn -> Elsa.Producer.ready?(connection) end)
+    :timer.sleep(@brod_init_sleep_ms)
```

Comment:
Producer.ready? only checks the Producer -- the problem we're having here is that the consumer isn't always _totally_ started up before we produce the test message it's supposed to pick up.  This is a problem in several places, and we've already got a ticket to investigate it: https://simplifi.atlassian.net/browse/INT-11108